composer/src/composer.rs

```diff
@@ -1695,4 +1690,21 @@ impl ComposeTest {
         let exit_code = self.docker.inspect_exec(&exec.id).await?.exit_code;
         Ok((exit_code, response))
     }
+
+    /// Print logs for all containers.
+    pub fn print_all_logs(&self) {
+        self.containers.keys().for_each(|name| {
+            self.print_log(name);
+            tracing::error!("");
```

Comment:
Do we need this extra log?

Reply:
If it's for padding, then maybe would be better a println! ?

Reply:
yes indeed